### PR TITLE
feat(ui): add login card component for portfolio page

### PR DESCRIPTION
**Pull request by yhabib**
# Motivation

The portfolio page needs a dedicated login card to allow the user to login from this page.

Ticket [NNS1-3510](https://dfinity.atlassian.net/browse/NNS1-3510)

https://github.com/user-attachments/assets/1c551e5e-45f1-44e6-ac44-a9076a1adf24

# Changes

- Creates new LoginCard component with authentication form

# Tests

- Not necessary as there is no logic

# Todos

- [ ] Add entry to changelog (if necessary).
Not necessary

Prev. PR: #6069

[NNS1-3510]: https://dfinity.atlassian.net/browse/NNS1-3510?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: frontend/src/lib/components/portfolio/LoginCard.svelte [hunk 1]
```diff
@@ -0,0 +1,84 @@
+<script>
+  import { i18n } from "$lib/stores/i18n";
+  import { IconAccountsPage } from "@dfinity/gix-components";
+  import SignIn from "../common/SignIn.svelte";
```
Comment (mstrasinskis) [inline frontend/src/lib/components/portfolio/LoginCard.svelte:4]: Would be nice to start imports here and on the next line from `$lib/...`?

File: frontend/src/lib/components/portfolio/LoginCard.svelte [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+        "icon description"
+        "icon action";
+      gap: var(--padding-3x);
+      padding: var(--padding-3x) var(--padding-4x);
+    }
+    h2 {
+      grid-area: title;
+      margin: 0;
+      padding: 0;
+      margin-bottom: calc(var(--padding) * -1);
```
Comment (mstrasinskis) [inline frontend/src/lib/components/portfolio/LoginCard.svelte:45]: Nitpick: not really critical, but can it be solved w/o negative margins. Maybe `grid-template-rows`?

File: frontend/src/lib/components/portfolio/LoginCard.svelte [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+      @include media.min-width(medium) {
+        margin-bottom: calc(var(--padding-2x) * -1);
+      }
+    }
+    p {
+      grid-area: description;
+      margin: 0;
+      padding: 0;
+      @include media.min-width(medium) {
+        grid-area: description;
```
Comment (mstrasinskis) [inline frontend/src/lib/components/portfolio/LoginCard.svelte:55]: Overriding the grid-area for the breakpoints seems redundant. Same for other areas.

File: frontend/src/lib/components/portfolio/LoginCard.svelte (hunk 2 above)
Comment (yhabib) [reply frontend/src/lib/components/portfolio/LoginCard.svelte]: I simplified the layout by introducing a div container that allows for more precise control over the gaps, eliminating the need for negative margins.

File: frontend/src/lib/components/portfolio/LoginCard.svelte [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+    display: grid;
+    grid-template-columns: auto 1fr;
+    grid-template-areas:
+      "icon content"
+      "action action";
+    gap: var(--padding-2x);
+    padding: var(--padding-2x);
+
+    @include media.min-width(medium) {
+      grid-template-columns: auto 1fr;
```
Comment (mstrasinskis) [inline frontend/src/lib/components/portfolio/LoginCard.svelte:35]: Seems overriding the grid-template-columns and grid-template-areas is a redundancy after recent changes.

File: frontend/src/lib/components/portfolio/LoginCard.svelte [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+    @include media.min-width(medium) {
+      grid-template-areas:
+        "icon content"
+        "icon action";
+      gap: var(--padding-3x);
+      padding: var(--padding-3x) var(--padding-4x);
+    }
+
+    .content {
+      h2,
```
Comment (yhabib) [inline frontend/src/lib/components/portfolio/LoginCard.svelte:43]: I need to reset this default styles

File: frontend/src/lib/components/portfolio/LoginCard.svelte (hunk 2 above)
Comment (mstrasinskis) [reply frontend/src/lib/components/portfolio/LoginCard.svelte]: Thanks.

**Review by mstrasinskis — APPROVED** [review]
LGTM, thanks!

**Pull request merged**